fedimint-core/src/iroh_prod.rs

```diff
@@ -1,7 +1,11 @@
+//! We want to provide our own infrastructure to be independent of n0, but for
+//! now we don't use the discovery part of it pending figuring out some
+//! performance regressions.
+
 /// The Iroh/Pkarr DNS server hosted by Fedimint project
-pub const FM_DNS_PKARR_RELAY_PROD: [&str; 2] = [
-    "https://dns.irohdns-eu-01.dev.fedimint.org/pkarr",
-    "https://dns.irohdns-us-01.dev.fedimint.org/pkarr",
+pub const FM_DNS_PKARR_RELAY_PROD: [&str; 0] = [
+    // "https://dns.irohdns-eu-01.dev.fedimint.org/pkarr",
+    // "https://dns.irohdns-us-01.dev.fedimint.org/pkarr",
 ];
 
 /// The Iroh relays hosted by Fedimint project
```

Comment:
Do we want to just keep our relays? Did you test with them @bradleystachurski? We only really talked about DNS issues.